caps/static/caps/js/main.js

```diff
@@ -312,6 +312,32 @@ if ($selectedArray.every( (val, i, arr) => val === arr[0])) {
 }
 // END ADVANCED FILTER ON COUNCIL LIST PAGE
 
+// OPEN DIV BUTTON
+/* Usage:
+<button type="button" class="btn btn--open-div">
+    Filter by authority type and others...
+</button>
+<div class="open-target"></div>
+
+In a case by case, the open-target should have the neccesary display property
+By default it will be block
+*/
+$('.btn--open-div').each(function(){
+    var $btn = $(this);
+    var $target = $btn.closest.next('.open-target');
+
+    $btn.text('Show more');
+    if ( $btn.is('.open') ) {
+        $btn.removeClass('open');
+        $target.removeClass('open');
+        $btn.text($defaultButtonText);
+    } else {
+        $btn.addClass('open');
+        $target.addClass('open');
+        $btn.text($closeButtonText);
+    }
+});
+// END OPEN DIV BUTTON
```

Comment:
I'm not sure I understand what this is for?

Reply:
Also, it feels like this is in contrast to the block that opens the div if there's a filter selected.

Reply:
You are right, those functions were conflicting with each other. So I ended up renaming the variables for the `#button-advanced-filter`. This one also uses different button text, so the code for `.btn--open-div` with the "Show more" and "Show less" text won't suffice. 

[The [CAPE] Added button to open div](https://github.com/mysociety/caps/pull/415/commits/a9bb96280d6a2676f7196646ee3d73597224c23d) was included so we can have a button we can reuse, not only for the .scorecard-table, but anywhere on the website.